src/engine.rs

```diff
@@ -463,15 +464,19 @@ impl Engine {
             .map(|result| Address(result.as_slice().try_into().unwrap()))
     }
 
-    pub fn register_token(&mut self, erc20_token: &[u8], nep141_token: &[u8]) {
+    pub fn register_token(&mut self, erc20_token: &[u8], nep141_account_id: &str) {
```

Comment:
This should take `nep141_account_id` as a `String`.

Reply:
I'd prefer having `AccountId` which is an alias to `String` right now. But in this case, I get a warning (which will result in error in CI) on clippy to use `&str` instead. Do you think I should force clippy to bypass this check?

Reply:
No, for that kind of check I wouldn't want to bypass it.